recipes/meshplex/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "meshplex" %}
+{% set version = "0.13.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/nschloe/meshplex/archive/v{{ version }}.tar.gz
+  sha256: ceadd26da8c4bfdd4e543f48944b43a5c38ad4faa55563330651fb6d6f6ac765
+
+build:
+  number: 0
+  skip: true  # [py==38]
```

Comment:
Should this be `>=py38`? Or does it work with `py39`?

Reply:
Actually it does work with py38, see https://github.com/nschloe/meshplex/blob/master/setup.cfg#L35

Reply:
OK. I have removed that